uk.co.compucorp.civicrm.hrcore/gulp/tasks/test.js

```diff
@@ -31,7 +31,7 @@ module.exports = [
       if (utils.canCurrentExtensionRun('test')) {
         extPath = utils.getExtensionPath();
         watchPatterns = utils.addExtensionCustomWatchPatternsToDefaultList([
-          path.join(extPath, '**', 'test/**/*.spec.js'),
```

Comment:
Would that mean that we will not be able to TDD the files belonging to the old structure?

Reply:
yes it does 
![anim](https://user-images.githubusercontent.com/1642119/41731321-1f4bebd0-754c-11e8-9abb-f0cceb44397a.gif)